src/Raven.Studio/typescript/models/database/tasks/ongoingTaskModel.ts

```diff
@@ -22,8 +23,19 @@ abstract class ongoingTaskModel {
         
         return (preferredMentor && currentNode) ? preferredMentor !== currentNode : false;
     });
+    
+    abstract get studioTaskType(): StudioTaskType;
+
+    static getStudioTaskType(taskListItem: Raven.Client.Documents.Operations.OngoingTasks.OngoingTask): StudioTaskType {
+        if (taskListItem.TaskType === "QueueEtl") {
+            const task = (taskListItem as Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskQueueEtlListView);
+            return task.BrokerType === "Kafka" ? "KafkaQueueEtl" : "RabbitQueueEtl";
```

Comment:
let's do switch here to throw when we add new broker type.

Reply:
done